### ISSUE TITLE
accountData folder is very vague and why with the client seed

### ISSUE THREAD

**Issue by John-Ivar-Flatmo** [WowLegacyCore/HermesProxy#235]
accountData folder is very vague as no folder is named anything like that, the closest is in WTF but then where does it go, i assume the entire WTF folder and im guessing from the original classic, but i don't know.

witch folder is it
what client does it belong to
does it even belong to a client, account atleast kinda indicates that it probably does
why do i have to copy it
where do i have to copy it to
is it supposed to remain in its parent folder
etc
point is i have no **** clue what folder it is so i just copied all the folders that may kinda indicate that its the right one, and i have no way to verify if i did it right since i have some probably unrelated issue

basically just specify witch folder that is idealy both in the readme and maybe info.txt perhaps even DONT_FORGET_TO_COPY_YOUR_AccountData_FOLDER

ffs while i wrote this i realized its probably the modern client and probably just so that Arctium can load it, in witch case why with the static seed thats also very unclear, and that might legit be my problem.

**Comment by ratkosrb** [comment]
AccountData is in the HermesProxy directory, not part of the client. Static auth seed is an option in the Arctium launcher, that patches seed the client uses for encryption when you login, so that you can successfully connect to the Realm socket of HermesProxy. If you don't use the static auth seed option in the launcher, then you have to set the correct seed for your build in the HermesProxy config. Don't ask how to find the real auth seed for your client, that requires you to be an expert in WoW reverse engineering, which is why the instructions say to use the static auth seed option in the launcher.

**Comment by John-Ivar-Flatmo** [comment]
i appreciate the response but this clarifies very little.

so the auth seed is required "because" ##im guessing modern client requires it otherwise the hard requirement makes no sense and should be optional

and the accountData is either the build CSV folder or the source HermesProxy folder,
i dont konw why it wuld be the source folder but you specified HermesProxy directory
if its the CSV then why does it not just say REMEMBER TO COPY CSV FOLDER
if its the HermesProxy  then why is it not included in build

**Comment by 0blu** [comment]
The `AccountData` holds character specific settings that cannot be requested from the server but are required by the modern client.
For example things like quest progression cannot be queried from a 1.12 server but some 1.14 AddOns require this feature.

Just start HermesProxy and ignore this `AccountData` folder. **It is only necessary if you want to upgrade to a newer HermesProxy version**.

If you play on Everlook and don't want to deal with this stuff you can use the hassle free [WinterspringLauncher](https://github.com/0blu/WinterspringLauncher).

**Comment by John-Ivar-Flatmo** [comment]
il just clarify i need this information because im hosting not playing.
thus i need actual information not just oh don't worry about it.
i simply haven't the luxury to not worry about it.
And i only mention it so that peaple like me don't have to deal with troubleshooting issues due to lack of information.

so i ask agein
wich folder is the AccountData? is it literaly the source code account data folder?
and doe's the modern client actualy need to send/ recive client proxy?
if not why in all that is great and good is it required by HermesProxy

**Comment by insunaa** [comment]
Why do you expect people to help you if you are so rude?
ratkosrb and 0blu have explained *exactly* what you asked, but you keep asking the same dumb questions.

**Comment by MaxtorCoder** [comment]
Everything got explained by @0blu and @ratkosrb, re-read the messages. :)
